pkg/config/shuttleconfig.go

```diff
@@ -6,7 +6,7 @@ import (
 	"path"
 
 	"github.com/lunarway/shuttle/pkg/ui"
-	yaml "gopkg.in/yaml.v2"
+	"gopkg.in/yaml.v2"
```

Comment:
What is the purpose of this change?

Reply:
I think that dependency optimizer did that, due to the package is actually called "yaml" which makes this redundant.